src/QUICServer.ts

```diff
@@ -346,8 +346,7 @@ class QUICServer {
           force?: boolean;
         } = {},
   ) {
-    const address = utils.buildAddress(this.socket.host, this.socket.port);
-    this.logger.info(`Stop ${this.constructor.name} on ${address}`);
+    this.logger.info(`Stop ${this.constructor.name}`);
```

Comment:
I think this should still exist... but as you said, if the socket were stopped prior to the server, then it cannot access the `host` and `port` parameters anymore.

We know that this would actually be considered a runtime error.

So instead of doing this, what we can do is check if the socket is running, and if it is, get the `address` and if it isn't running, don't use the address.

The same issue has to be applied on `QUICClient` too as it has the same problem.

Reply:
Fixed this now in d5f6519ed01f7e0aa87bd0a9090a0ab299e21451